.travis.yml

```diff
@@ -5,6 +5,8 @@ services:
 
 language: java
 before_install:
+  - docker run --name dbfitdb2 -d -p 50000:50000 -v "`pwd`/dbfit-java/db2/src/integration-test/resources:/var/dbfit/dbfit-java/db2/src/integration-test/resources" -e DB2INST1_PASSWORD=db2inst1-pwd -e LICENSE=accept ibmcom/db2express-c:latest db2start
```

Comment:
You shouldn't rely on hard-coded paths, you only rely on having current dir in the project root.
Anyway - separate script is perhaps not worth for that. Lines can be wrapped like I did in the Oracle polling script. Also these paths seem excessively long to me (do we really need so long paths?)